flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -100,6 +100,12 @@ under the License.
 			<version>${project.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-sql-parser</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
I would suggest to add this when we need it.

Reply:
I already need this dependency now for the bridge tool class `SqlExecutableStatement`.

Reply:
Oh, I commented in the wrong file. I mean the pom.xml in `flink-table-planner`.

Reply:
Thanks, removed